eng/java.gdnbaselines

```diff
@@ -0,0 +1,2183 @@
+{
+  "version": "1.0.0",
+  "baselines": {
+    "baseline": {
+      "name": "baseline",
+      "createdDate": "2021-01-22 03:39:03Z",
+      "lastUpdatedDate": "2021-01-22 03:39:03Z"
+    }
+  },
+  "results": {
+    "a69fb9406f784dbf3b85e1a190ae4cc72712e321df5c924fd7eb4497497e44ec": {
```

Comment:
Similar to what you are doing with .NET team can you look to see if there is a way to maybe add a set of placeholder values instead a bunch of baselined inclusions? You can do that a s a follow up though.

Reply:
That's the suppression rule. If developer is certain about the allowlist of the key/secret, we can add them as placeholder in suppression rule. For this PR, it is to enable the cred scan first

Reply:
OK once we get the suppressions worked out we should be sure to clean out these baseline files.